<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridOpponent"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridPlayer"/>
   </widget>
   <widget class="QLCDNumber" name="lcdPlayer">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="numDigits">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="slabelOpponent">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Opponent:</string>
    </property>
   </widget>
   <widget class="QLabel" name="slabelPlayer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdOpponent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="numDigits">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStatus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textInfoDisplay">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>441</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textMessageInput">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSendMessage">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatsPlayer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatsOpponent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="slabelGameStats">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Game Stats:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>textMessageInput</sender>
   <signal>returnPressed()</signal>
   <receiver>buttonSendMessage</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
